src/views/Samples.vue

```diff
@@ -42,22 +47,23 @@ export default {
   created() {
   },
   methods: {
-    async createLibraries () {
+    async createLibraries (selectedEnzymeId) {
       try {
-        await this.createLibrariesInTraction()
+        await this.createLibrariesInTraction(selectedEnzymeId)
       } catch (error) {
         // log error
       } finally {
         this.showAlert
       }
     },
-    async createLibrariesInTraction () {
-      let sample_ids = []
+    async createLibrariesInTraction (selectedEnzymeId) {
```

Comment:
Can we add a TODO to have a look at the processing in the test and see if it signifies an issue elsewhere.